commit 19d97ac5aabec2e253fd40d110ecbe326040d917
Author: Chen Hanxiao <chenhx.fnst@fujitsu.com>
Date:   Fri Nov 22 10:18:25 2024 +0800

    nfsd: trace: remove redundant stateid even deleg_recall
    
    Since commit e56dc9e2949e ("nfsd: remove fault injection code") remove
    all nfsd_recall_delegations codes,
    we don't need trace_nfsd_deleg_recall any more.
    
    Signed-off-by: Chen Hanxiao <chenhx.fnst@fujitsu.com>
    Signed-off-by: Chuck Lever <chuck.lever@oracle.com>

diff --git a/fs/nfsd/trace.h b/fs/nfsd/trace.h
index 696c89f68a9e..ad2c0c432d08 100644
--- a/fs/nfsd/trace.h
+++ b/fs/nfsd/trace.h
@@ -626,7 +626,6 @@ DEFINE_STATEID_EVENT(open);
 DEFINE_STATEID_EVENT(deleg_read);
 DEFINE_STATEID_EVENT(deleg_write);
 DEFINE_STATEID_EVENT(deleg_return);
-DEFINE_STATEID_EVENT(deleg_recall);
 
 DECLARE_EVENT_CLASS(nfsd_stateseqid_class,
 	TP_PROTO(u32 seqid, const stateid_t *stp),